### PR TITLE
Add Profile components in ProfileStep

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?
This is the last ~piece~ PR needed for the profile components actually be included in the new Checkout.

[Related clubhouse story](https://app.clubhouse.io/vtex/story/32825/criar-componentes-de-profile-form-e-profile-preview).

#### How should this be manually tested?
[Workspace](https://chkio--checkoutio.myvtex.com/checkout)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by jeffersontuc — APPROVED** [review]

File: react/Step.tsx [hunk 1]
```diff
@@ -105,17 +105,25 @@ const Step: React.FC<StepProps> = ({
       <div
         className={classNames(
           handles.stepContentWrapper,
-          'mt4 mb6 mb7-ns w-100 flex'
+          'mb6 mb7-ns w-100 flex',
+          {
+            'mt4 mt3-ns': !active,
+            'mt4 mt5-ns': active,
```
Comment (thiagomurakami) [inline react/Step.tsx]: `mt4` is always added

File: react/Step.tsx [hunk 2]
```diff
@@ -105,17 +105,25 @@ const Step: React.FC<StepProps> = ({
...
       >
         <div
           style={{ width: (index ?? 0) < lastIndex ? 1 : 0 }}
           className={classNames(
             handles.stepDivider,
-            'dn db-ns bg-muted-4 nt4 nb6 nb7-ns mh5'
+            'dn db-ns bg-muted-4 nb6 nb7-ns mh5',
+            {
+              'nt4 nt3-ns': !active,
+              'nt4 nt5-ns': active,
```
Comment (thiagomurakami) [inline react/Step.tsx]: `nt4` is always added

File: manifest.json [hunk 3]
```diff
@@ -14,7 +14,9 @@
     "vtex.styleguide": "9.x",
     "vtex.css-handles": "0.x",
     "vtex.checkout-container": "0.x",
-    "vtex.checkout-payment": "0.x"
+    "vtex.checkout-payment": "0.x",
+    "vtex.checkout-profile": "0.x",
+    "vtex.order-manager": "0.x"
```
Comment (thiagomurakami) [inline manifest.json]: ⛏ Sorry about nitpicking about this, but could you sort the dependencies alphabetically? 😬

**Review by thiagomurakami — CHANGES_REQUESTED** [review]

File: react/Step.tsx (hunk 1 above)
Comment (lucasecdb) [reply react/Step.tsx]: nice catch!

File: manifest.json (hunk 3 above)
Comment (lucasecdb) [reply manifest.json]: sure thing!

**Review by thiagomurakami — APPROVED** [review]
LGTM

**Pull request merged**